forceng.js

```diff
@@ -305,7 +356,54 @@ angular.module('forceng', [])
      *  data:  JSON object to send in the request body - Optional
      */
     function request(obj) {
-
+        // NB: networkPlugin will be defined only if login was done through plugin and container is using Mobile SDK 5.0 or above
+        if (false && networkPlugin) { 
```

Comment:
This if branch will never evaluate to true - is that intentional and if so, should we just remove it?